src/test/java/com/keeper/homepage/domain/member/application/MemberServiceTest.java

```diff
@@ -51,4 +54,37 @@ void setUp() {
           .hasMessageContaining(MEMBER_CANNOT_FOLLOW_ME.getMessage());
     }
   }
+
+  @Nested
+  @DisplayName("회원 타입 변경 테스트")
+  class UpdateMemberTypeTest {
+
+    private Member member, other;
+
+    @BeforeEach
+    void setUp() {
+      member = memberTestHelper.generate();
+      other = memberTestHelper.generate();
+    }
+
+    @Test
+    @DisplayName("회원 타입 변경에 성공해야 한다.")
+    public void 회원_타입_변경에_성공해야_한다() {
+      Set<Long> memberSet = new HashSet<>();
+      memberSet.add(member.getId());
+      memberSet.add(other.getId());
+
+      memberService.updateMemberType(memberSet, 3);
+
+      em.flush();
+      em.clear();
+
+      Member findMember = memberRepository.findById(member.getId()).orElseThrow();
+      Member findOtherMember = memberRepository.findById(other.getId()).orElseThrow();
+
+      assertThat(findMember.getMemberType().getType()).isEqualTo(MemberTypeEnum.휴면회원);
+      assertThat(findOtherMember.getMemberType().getType()).isEqualTo(MemberTypeEnum.휴면회원);
```

Comment:
```suggestion
      assertThat(findMember.getMemberType().getType()).isEqualTo(휴면회원);
      assertThat(findOtherMember.getMemberType().getType()).isEqualTo(휴면회원);
```
Nit: static import를 사용하면 간결해질 것 같아요 ㅎㅎ

Reply:
감사합니다~~ 수정했습니닷!!